packages/boba/gateway/src/services/networkService.js

```diff
@@ -379,7 +379,7 @@ class NetworkService {
         //ok, that's reasonable
         //rinkeby, L1
         this.L1orL2 = 'L1'
-      } else if (masterSystemConfig === 'rinkeby' && network.chainId === 420) {
+      } else if (masterSystemConfig === 'rinkeby' && network.chainId === 28) {
```

Comment:
Is this number 28 fixed or it's going to be change over time. 

If that's the case we should load it from .env or some config file.

Reply:
```
      const nw = getAllNetworks()

      const L1ChainId = nw[masterSystemConfig]['L1']['chainId'];
      const L2ChainId = nw[masterSystemConfig]['L2']['chainId'];
     
      then we can compare on network.chainId === L1ChainId like way.

```

Reply:
https://github.com/omgnetwork/optimism-v2/pull/69